lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -679,6 +680,11 @@ instance FromText Direction where
 instance ToText Direction where
     toText = toTextFromBoundedEnum SnakeLowerCase
 
+-- | @SignedTxBinary@ is a serialised transaction that is ready to be submited
+-- to the node.
+newtype SignedTxBinary = SignedTxBinary { getSignedTxBinary :: ByteString }
+    deriving (Show, Eq, Generic)
```

Comment:
What about calling this `SealedTx` or simply `SignedTx` ?

Reply:
I liked the inclusion of `Binary` in the name to convey that "there is nothing we can do with this but submit it".

I suppose `SealedTx` conveys the same message 🤔

Reply:
Well, IMO having `Binary` in the name makes it redundant with the type that already conveys this information. Plus, what's the point of hiding the bytestring behind a "semi-opaque" newtype if we shout to everyone that it's holding a bytestring :sweat_smile: ...

Reply:
> Plus, what's the point of hiding the bytestring behind a "semi-opaque" newtype if we shout to everyone that it's holding a bytestring

We'll be saying "This is basically a byte string, but also a particular type of ByteString. Be careful."

That sounds fine to me.

`SealedTx` is more abstract. It's less clear (just from the name) whether:
- It can be serialised to binary easily
- It is specific to `jormungandr` or not

`SealedTx` _is_ catchy though… Partly taking your word for it (but I'm torn) -> d9ad073715f3aca45352f5a912460e97ae10ed59